Form/Type/ColorSelectorType.php

```diff
@@ -48,7 +48,15 @@ public function configureOptions(OptionsResolver $resolver)
                 Colors::WHITE,
                 Colors::YELLOW,
             ),
-        ));
+        );
+
+        // SF >2.8
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $defaults['choices'] = array_unique(array_flip(Colors::getAll()));
```

Comment:
> Hmm… wouldn't that be a BC-break? If yes, they should be added back to the flipped array, right ?

Good point... The colors are the same but with different names. We could remove than or make an:
```php
        // NEXT_MAJOR: Remove this "if" (when requirement of Symfony is >= 2.8)
        if($isInverted) {
            @trigger_error(
                'The $isInverted argument of '.__METHOD__.' is deprecated since 3.3, to be removed in 4.0.',
                E_USER_DEPRECATED
            );
            $colors[self::FUCHSIA] = 'fuchsia';
            $colors[self::CYAN] = 'cyan';
            return array_flip($colors);
        }
```